x-pack/plugins/cloud_security_posture/public/pages/vulnerability_dashboard/vulnerability_table_panel_section.tsx

```diff
@@ -159,11 +159,15 @@ export const VulnerabilityTablePanelSection = () => {
               defaultMessage: 'CVSS',
             }
           ),
-          render: (cvss: { score: number; version: string }) => (
-            <EuiLink onClick={() => onCellClick({ 'vulnerability.score.base': cvss.score })}>
-              <CVSScoreBadge version={cvss.version} score={cvss.score} />
-            </EuiLink>
-          ),
+          render: (cvss: { score: number; version: string }) => {
+            if (!cvss.score || !cvss.version) return null;
```

Comment:
I'm approving but was this agreed by product? if not, worth consulting. they might prefer some `unknown`/`undefined` tag or something. same for the vuln table itself.

Reply:
Thanks for highlighting this! I'm bringing it up in this comment https://github.com/elastic/security-team/issues/7146#issuecomment-1682475106 and think we need to specifically work on the UX of missing fields. Without this change, the behavior is different between dashboards and data grid (in the dashboard the empty green badge is being displayed as `CVSScoreBadge` doesn't handle this case itself, while in the data grid, we already have guards for a missing score in grid cells), so the question is if it's ok to move forward with the simplest solution. @nick-alayil do you see any problem with displaying empty cells consistently when the score is missing on the vulnerability document

Reply:
>@nick-alayil do you see any problem with displaying empty cells consistently when the score is missing on the vulnerability document

This is totally acceptable at the moment since our primary goal is to ensure that all identified CVEs are presented to the end user, accompanied by the resource ID. We can focus on enhancing the UX for the default values in the future.